test/test_helper.rb

```diff
@@ -67,6 +67,7 @@ def without_metadata_denormalisation(*klasses, &block)
 
   def stub_register_published_content
     WebMock.stub_request(:put, %r{\A#{PANOPTICON_ENDPOINT}/artefacts/})
+    WebMock.stub_request(:post, %r{search.dev.gov.uk/documents})
```

Comment:
In hindsight this does seem strange, but this changelog entry would suggest the 'test.gov.uk' behaviour is no longer present: https://github.com/alphagov/plek/blob/e57c986c8163257267197590ec0211f9ddb1da46/CHANGELOG.md#to-upgrade